src/NUnitFramework/framework/Internal/Results/TestResult.cs

```diff
@@ -511,7 +511,7 @@ public void RecordException(Exception ex)
             if (AssertionResults.Count > 0 && result.ResultState == ResultState.Error)
             {
                 // Add pending failures to the legacy result message
-                Message += CreateLegacyFailureMessage();
+                Message += " " + CreateLegacyFailureMessage();
```

Comment:
See my comment on the issue. This is obviously better than what we have, but I'm not sure it's enough.

Reply:
I agree with @ChrisMaddock that the simpler fix is best. Personally, I'd expect to see the legacy failure message under the original message with a blank line separating, i.e. 2 newlines rather than a space added.

Reply:
I would like to see two newlines as well.